nuxeo-coldstorage/src/main/resources/OSGI-INF/coldstorage-events-contrib.xml

```diff
@@ -3,34 +3,44 @@
 
   <extension target="org.nuxeo.ecm.core.scheduler.SchedulerService" point="schedule">
     <schedule id="checkAvailabilityOfBlobsBeingRetrieved">
-      <!-- every hour at the 7th minute -->
-      <cronExpression>0 7 * ? * * *</cronExpression>
+      <!-- every 5 minute -->
+      <cronExpression>0 0/5 * ? * * *</cronExpression>
       <event>checkColdStorageContentAvailability</event>
     </schedule>
   </extension>
```

Comment:
Later you could extract this part to `nuxeo.default` as we did for Aspera which will simplify the CI/CD conf :)

Reply:
Sorry
I did it

Reply:
@doulba thanks,
as discussed it I think you want to revert this part. But if you think we should keep. To avoid the errors that we saw, the scheduler should be disabled for tests purposes [Here](https://github.com/nuxeo/nuxeo-aspera-connector/blob/master/nuxeo-aspera-core/src/test/resources/OSGI-INF/aspera-scheduler-contrib-test-disable.xml).